project/accounts/models.py

```diff
@@ -149,33 +97,41 @@ def profile_image_thumb_url(self):
 
         return "/static/img/no_image_md.png"
 
-    def __init__(self, *args, **kwargs):
-        super(Profile, self).__init__(*args, **kwargs)
-
     def save(self, *args, **kwargs):
         """Image crop/resize and thumbnail creation"""
 
         # New Profile image --
         if self.profile_image:
             self.resize_profile_image()
```

Comment:
So whenever we create or update a profile, we try to resize the profile image no matter whether the profile image is edited or not. This image operation should be done during the registration and updating of the profile.

This might be out of the scope of this PR but we should consider it in the future.

Reply:
Good call. Let's add it to the backlog as a separate task. That way, a contributor might take it.